scenarioo-docu-generation-example/.gitignore

```diff
@@ -1,4 +1,5 @@
 build
+out
```

Comment:
@adiherzog Why is this needed now?? I not understand it

Reply:
I had a wrong IntelliJ configuration that compiled into the out folder. So I will remove "out" again.